frontend/demo/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -41,25 +41,29 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   };
 
   return (
-    <form className={styles.container}>
```

Comment:
it was a form because of accesbility resons. If something looks like a form, it should be a form. This helps screen readers a lot. Not that we have a super focus on this, but we should at least try to do a few things into the right direction :D

Reply:
Thanks for the insight